chainer/backends/intel64.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import absolute_import
+
+import chainer
+from chainer.configuration import config
+
+
+_ideep_version = None
+_error = None
+
+try:
+    import ideep4py as ideep  # NOQA
+    _ideep_version = 0
+except ImportError as e:
+    _error = e
```

Comment:
I think it better to make `mdarray` always available (as in [cuda](https://github.com/niboshi/chainer/blob/d5cd05e596c80943f4259940682a5eab593ec055/chainer/backends/cuda.py#L53-L58)), so that we don't have to check `is_ideep_available` everywhere just for type testing?

Reply:
Fixed.